### PR TITLE
Updated scripts and symbolic links for dapp build

### PR DESCRIPTION
**Pull request by Count042**
This are some small fixes to allow deployment of the dapp.

### CODE REVIEW

File: web-ethereum/dapp2/build.sh [hunk 1]
```diff
@@ -10,6 +10,6 @@ $FLUTTER build web --web-renderer canvaskit
 
 # move js to hash named file
 uuid=$(uuidgen)
-sed -i '' "s/main.dart.js/${uuid}.main.dart.js/g" build/web/index.html
```
Comment (patniemeyer) [inline web-ethereum/dapp2/build.sh:13]: What do the empty quotes do here?

File: web-ethereum/dapp2/lib/l10n [hunk 2]
```diff
@@ -1 +1 @@
-/Users/pat/Desktop/OrchidProject/lab.orchid.com/orchid/app-flutter/lib/l10n
```
Comment (patniemeyer) [inline web-ethereum/dapp2/lib/l10n:1]: Good catch.

File: web-ethereum/dapp2/build.sh (hunk 1 above)
Comment (Count042) [reply web-ethereum/dapp2/build.sh:13]: I'm not sure, but it wouldn't work with the default linux sed.   I switched it out for double quotes which does work, and allows for the variable replacement.

**Review by patniemeyer — APPROVED** [review]
Looks great!  thanks.

**Pull request merged**